tests/postgres_tests/models.py

```diff
@@ -62,6 +62,10 @@ class NestedIntegerArrayModel(PostgreSQLModel):
     field = ArrayField(ArrayField(models.IntegerField()))
 
 
+class NestedNullableIntegerArrayModel(PostgreSQLModel):
+    field = ArrayField(ArrayField(models.IntegerField(null=True)))
```

Comment:
I don't think that we need a separate model, I would add 
```
field_nullable = ArrayField(ArrayField(models.IntegerField(null=True)))
```
to the `NestedIntegerArrayModel`.